src/App.tsx

```diff
@@ -15,48 +17,108 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+export enum SortType {
+  Alphabet = 'abc',
+  Length = 'length',
+  Default = '',
+}
```

Comment:
You can remove enum values as you don't use them into anywhere else

```suggestion
export enum SortType {
  Alphabet,
  Length,
  Default,
}
```

Reply:
agree)